openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -131,31 +137,35 @@ def setUpTestData(cls):
             title="User Outline Test Course!",
             published_at=datetime(2020, 5, 20, tzinfo=timezone.utc),
             published_version="5ebece4b69dd593d82fe2020",
-            sections=generate_sections(cls.course_key, [2, 1, 3])
+            sections=generate_sections(cls.course_key, [2, 1, 3]),
+            course_visibility=CourseVisibilities.PUBLIC
```

Comment:
Should this be testing `PUBLIC` or `PRIVATE` here? Aren't most courses `PRIVATE` by default?

Reply:
@pcockwell 
If you see the docstring of this test, it says
```
        """This outline is the same for everyone."""
```
Outline would be same for everyone only in case the course visibility is `PUBLIC`

Reply:
While that may be true, the dosctring was written before your changes to course visibility were added. The test itself is for a `simple_outline`, which to me, would indicate testing the most common, base case. Wouldn't that be `PRIVATE` instead of public?

Reply:
@pcockwell Yes, you are right that the most common, base case would be `PRIVATE`. I would fix it.